integration/testdata/custom_conf_app/buildpack.yml

```diff
@@ -0,0 +1,3 @@
+---
+nginx:
+  version: "mainline"
```

Comment:
Do we need a `buildpack.yml` file at all for this integration test? Support for it is deprecated so we probably don't want to create this file when we make new test fixtures

Reply:
No we don't need this file. I'll remove it.